packages/manager/src/components/NotificationMenu/NotificationMenu.tsx

```diff
@@ -1,75 +1,55 @@
 import * as React from 'react';
 import { useDispatch } from 'react-redux';
 import { makeStyles, Theme } from 'src/components/core/styles';
-import Drawer from 'src/components/Drawer';
+import Events from 'src/features/NotificationCenter/Events';
+import { NotificationData } from 'src/features/NotificationCenter/NotificationData/useNotificationData';
+import Notifications from 'src/features/NotificationCenter/Notifications';
 import useDismissibleNotifications from 'src/hooks/useDismissibleNotifications';
 import useNotifications from 'src/hooks/useNotifications';
 import usePrevious from 'src/hooks/usePrevious';
 import { markAllSeen } from 'src/store/events/event.request';
 import { ThunkDispatch } from 'src/store/types';
-import { NotificationData } from 'src/features/NotificationCenter/NotificationData/useNotificationData';
-import Events from 'src/features/NotificationCenter/Events';
-import Notifications from 'src/features/NotificationCenter/Notifications';
 
 const useStyles = makeStyles((theme: Theme) => ({
   root: {
-    '& .MuiDrawer-paper': {
-      [theme.breakpoints.up('md')]: {
-        width: 620,
-      },
-      overflowX: 'hidden',
-    },
-  },
-  notificationSectionContainer: {
-    '& > div': {
-      marginTop: theme.spacing(3),
-      marginBottom: theme.spacing(3),
-    },
-  },
-  actionHeader: {
-    display: 'flex',
-    paddingBottom: theme.spacing(),
-    justifyContent: 'flex-end',
-    borderBottom: `solid 1px ${theme.palette.divider}`,
-    marginBottom: theme.spacing(3),
+    padding: 20,
```

Comment:
Dark mode background color needs to be fixed. This suggested change seems to take care of it. 

![Screen Shot 2022-04-15 at 9 59 29 AM](https://user-images.githubusercontent.com/6440455/163579885-8951ba44-c9ec-48cd-af98-df4822e631ae.jpg)



```suggestion
    backgroundColor: theme.bg.bgPaper,
    padding: 20,
```